cypress/support/commands/application.js

```diff
@@ -83,3 +83,27 @@ Cypress.Commands.add(
     );
   },
 );
+Cypress.Commands.add(
+  'getPageData',
+  (service, pageType, variant = 'default', urlOverride) => {
```

Comment:
Given that there quite a few parameters to this function, can we pass parameters by name, instead of by order?

```suggestion
  ({ service, pageType, variant = 'default', urlOverride }) => {
```